akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala

```diff
@@ -766,7 +766,7 @@ private[http] object `Sec-WebSocket-Extensions` extends ModeledCompanion[`Sec-We
  */
 @InternalApi
 private[http] final case class `Sec-WebSocket-Extensions`(extensions: immutable.Seq[WebSocketExtension])
-  extends ResponseHeader {
+  extends RequestResponseHeader {
```

Comment:
LGTM, quoting spec that @ignasi35 mentions in ticket for fellow reviewers:

`It is initially sent from the client to the server, and then subsequently sent from the server to the client, to agree on a set of protocol-level extensions to use for the duration of the connection. `

Reply:
Also relevant from the spec but not tested or addressed in this PR:

> The |Sec-WebSocket-Extensions| header field MAY appear multiple times
> in an HTTP request (which is logically the same as a single
> |Sec-WebSocket-Extensions| header field that contains all values.
> However, the |Sec-WebSocket-Extensions| header field MUST NOT appear
> more than once in an HTTP response.

I made no effort in asserting that the header may appear several times and that values are preserved when that happens.

Reply:
I'm ok with not enforcing this strickly, would be costly on insertions